tests/integration/states/test_archive.py

```diff
@@ -265,10 +265,51 @@ def test_local_archive_extracted_with_skip_files_list_verify(self):
         expected_comment = ('existing source sum is the same as the expected one and '
                             'skip_files_list_verify argument was set to True. '
                             'Extraction is not needed')
+
+        # Clearing the minion cache at the start to ensure that different tests of
+        # skip_files_list_verify won't affect each other
+        self.run_function('saltutil.clear_cache')
```

Comment:
This is required due to cache limitation. Right now any test with skip_files_list_verify=True and source_hash_update=True will create a hash file in minion cache and other tests will use it when they shouldn't.

The alternative was to pass a different source for each test which sounds worse.